README.md

```diff
@@ -36,6 +36,12 @@ optional arguments:
   --log-level LOG_LEVEL
                         DEBUG, INFO, WARNING, ERROR, CRITICAL
 
+# Having difficulty using proxy.py? Report at:
+://github.com/abhinavsingh/proxy.py/issues/new
+```
+
+
+Test comments
+abcd efefete
```

Comment:
Naming is very hard, but i'd like to work with you with better naming.

Would you please share more about the decision making of this code change ?